scripts/build_image

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# set readable variables
+export JAVA_VERSION_TEMPLATE="$1"
+export NODE_VERSION_TEMPLATE="$2"
+export MBT_VERSION="$3"
+
+# set jvm and node version in Dockerfile
+if [ "$JAVA_VERSION_TEMPLATE" = "11" ]
+then
+	export JAVA_VERSION="-java11"
+	export NODE_VERSION="-node$(echo $NODE_VERSION_TEMPLATE |awk -F. '{printf "%d", $1}')"
+	cp Dockerfile_mbtci_java11 Dockerfile
+	sed  -i "s/NODE_VERSION_TEMPLATE/v${NODE_VERSION_TEMPLATE}/" Dockerfile
```

Comment:
are you planing on supporting multiple nodejs versions?
if not the `NODE_VERSION_TEMPLATE` may not be needed

Reply:
Yes, we plan to support multiple java and node version, the NODE_VERSION_TEMPLATE is needed.

The support plan is : https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=3138611365